src/ui/render.ts

```diff
@@ -14,6 +14,9 @@ import LightSheetHelper from "../../utils/helpers.ts";
 
 export default class UI {
   tableEl: Element;
+  lightSheetFormulaBarDom: HTMLElement;
+  lightSheetFormulaInput: HTMLInputElement;
+  selectedCellDisplay: HTMLElement;
```

Comment:
Remove the lightSheet prefix from the variable names.
example: make it formulaBarDom, not lightSheetFormulaBarDom

Reply:
In this issue, I just want to make a placeholder for selectedCellDisplay and  I think the functionality of it should be moved to a new issue?